.gitignore

```diff
@@ -0,0 +1,61 @@
+# Quarto build outputs
+_site/
+.quarto/
```

Comment:
Didn't we say we'd go with mkdocs-material?

Would recommend starting with a minimal `.gitignore` in case.